.github/workflows/build_test.sh

```diff
@@ -156,12 +156,6 @@ for args in "${ARGS[@]}"; do
         fatal "'meson compile' failed with '$args'"
     fi
 
-    for loader in build/src/boot/efi/*.efi; do
-        if sbverify --list "$loader" |& grep -q "gap in section table"; then
-            fatal "$loader: Gaps found in section table"
-        fi
-    done
```

Comment:
Don't we want to keep this test?

Reply:
Dunno really why I wanted to remove. It shouldn't happen anymore but also doesn't hurt to have. In fact, I made it stricter now.